microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/config/java/MicroProfileConfigASTValidator.java

```diff
@@ -105,15 +107,17 @@ public boolean visit(TypeDeclaration typeDeclaration) {
 		List modifiers = typeDeclaration.modifiers();
 		for (Object modifier : modifiers) {
 			if (modifier instanceof NormalAnnotation) {
-				try {
-					Expression prefixExpr = AnnotationUtils.getAnnotationMemberValueExpression(
-							(NormalAnnotation) modifier, CONFIG_PROPERTIES_ANNOTATION_PREFIX);
-					if (prefixExpr != null) {
-						currentPrefix = ((StringLiteral) prefixExpr).getLiteralValue();
+				if (AnnotationUtils.isMatchAnnotation((IAnnotation) modifier, CONFIG_PROPERTIES_ANNOTATION_PREFIX)) {

```

Comment:
I think yiu should use CONFIG_PROPERTIES_ANNOTATION no?

Reply:
Also spoke to @JessicaJHee about casting :)